server/config/config.js

```diff
@@ -2,6 +2,7 @@ const config = {
   mongoURI: process.env.MONGO_URI || "mongodb://localhost",
   mongoPort: process.env.MONGO_PORT || "27017",
   mongoDB: process.env.MONGO_DB || "mail-sender-dev",
+  secretOrKey: "secret"
```

Comment:
Given this is pretty important when we deploy, let's get this from process.env.SECRET even if it's just secret

Reply:
I don't like the name `secretOrKey`, the reason JWT middleware uses that, is because it could be a secret or a key, so we should have one here (basically a secret, so maybe a name like `appSecret`)